scripts/git_version_info.sh

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+#Store the version information from the .git into a file so it can be used later at runtime after removing git
+
+GIT_REPO=$(git config --local remote.origin.url)
+GIT_COMMIT=$(git rev-parse HEAD)
+GIT_BRANCH=$(git name-rev "$GIT_COMMIT" --name-only)
+
+echo "export GIT_REPO=$GIT_REPO
+export GIT_COMMIT=$GIT_COMMIT
+export GIT_BRANCH=$GIT_BRANCH" > /etc/git.version
```

Comment:
Nice to not have it at the root, but this doesn't really fit with the `data_validation` stuff already in `scripts` that much. I'm okay with this, just thinking about optimal file organization.

Reply:
Yeah, I saw we had a scripts directory so this *seemed* like the better place for it. I'll leave this in there for now as I'd like to have the root a little lighter. It's **possible** it'd fit better in dockerfiles since it's more just related to the build.